docs/sdks/go/index.md

```diff
@@ -173,6 +171,9 @@ if err != nil {
 }
 fromAddr := myAddress
 toAddr := "GD64YIY3TWGDMCNPP553DZPPR6LDUSFQOIJVFDPPXWEG3FVOJCCDBBHU5A"
+
+closeToAddr := "HZ57J3K46JIJXILONBBZOHX6BKPXEM2VVXNRFSUED6DKFD5ZD24PMJ3MVA"
```

Comment:
missed this one?

Reply:
Ya, i had deleted it, then the merge commit pt it back. all set now.